### DISCUSSION TITLE
tinyUSB stuck STM32f4

### DISCUSSION THREAD

**Discussion by huynhphucloi** [hathach/tinyusb#1672] [General]
Hi. I have added tinyUSB to my project STM32F407. but it seems to be stuck here. and i tried many ways but still can't solve
config:
<img width="856" alt="Screen Shot 2022-10-10 at 08 56 44" src="https://user-images.githubusercontent.com/64252151/194790608-372d5591-2845-4c74-a6d3-807d700c4952.png">

stuck here:
<img width="655" alt="Screen Shot 2022-10-10 at 08 57 41" src="https://user-images.githubusercontent.com/64252151/194790623-a2222ee6-5a86-4858-a526-a36eab8fc825.png">

**Comment by hathach** [comment]
- which example that you run, and on which board. Please try one of supported boards first if possible
- what is your debug log

**Reply by huynhphucloi** [reply]
The error appeared because I used FS but used the config of HS port 1 changed as shown in the picture and it was working properly. Thank you for your interest
<img width="808" alt="Screen Shot 2022-10-10 at 15 50 39" src="https://user-images.githubusercontent.com/64252151/194830317-cc2f4bde-31e4-4bfb-bb7c-57e9031cddec.png">

<img width="805" alt="Screen Shot 2022-10-10 at 15 51 39" src="https://user-images.githubusercontent.com/64252151/194830397-707045a4-50aa-4ed6-905d-ab933830da53.png">
